versioned_docs/version-8.3/self-managed/zeebe-deployment/operations/resource-planning.md

```diff
@@ -1,7 +1,15 @@
 ---
 id: resource-planning
 title: "Resource planning"
-keywords: ["backpressure", "back-pressure", "back pressure"]
+keywords:
```

Comment:
Did we want to backport these changes, and include in `8.4`, `8.5`, `next` as well?

Reply:
It came from 8.3 originally and I forwarded it to the latest versions plus next.